test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/DropInvalidMetadataTests.cs

```diff
@@ -27,7 +27,7 @@ public class DropInvalidMetadataTests : IClassFixture<EnableDropInvalidDicomJson
     public DropInvalidMetadataTests(EnableDropInvalidDicomJsonMetadataHttpIntegrationTestFixture<Startup> fixture)
     {
         EnsureArg.IsNotNull(fixture, nameof(fixture));
-        _client = fixture.GetDicomWebClient();
+        _client = fixture.GetDicomWebClient(DicomApiVersions.V2);
```

Comment:
Do we need a test that checks if the client supports `v2` but the client sends `v1` that the expected behavior happens? Also, I assume we have a test somewhere that does the opposite (e.g. makes sure if you use `v2` with it disabled, that you get a failure?